### PR TITLE
Create UCSanDiego.ts

### PR DESCRIPTION
**Pull request by trulyronak**
## University of California San Diego (no comma)
As we move forward to strengthen the brand, we will no longer use a comma when referencing the University of California San Diego. Our goals are to avoid the multiple variations currently used for naming the campus—University of California [at or in, or with the comma] San Diego—and to foster consistency, so use this name in all print and online applications.

## “UC San Diego” not “UCSD”
Using UC San Diego in place of the UCSD acronym better identifies our campus both locally and nationally. There’s confusion among San Diego higher education institutions because of similar acronyms—UCSD, USD, and SDSU—which we eliminate by using UC San Diego. Additionally, this naming convention is consistent with other campuses in the University of California system, such as UC Irvine, UC Riverside, UC Santa Barbara, UC Davis, and so on.

## Citing the Campus
In all communications efforts, please follow this protocol whenever possible:

First reference, spell out the name: University of California San Diego.
Subsequent references and in headlines, use the abbreviated version of the name: UC San Diego (not UCSD).
You can also use other words such as “campus” or “institution” to refer to UC San Diego in various communications efforts.
Name Use Reference
For information about the use of the University of California and UC San Diego names, please see http://go.ucsd.edu/1U7dzpO.

## Name Use Policy and Protocol
For information about policy and protocol related to the use of the University of California and UC San Diego name, please see the Use of the University of California Name guidelines.

### CODE REVIEW

File: src/commands/UCSanDiego.ts
Comment (StormFireFox1) [inline src/commands/UCSanDiego.ts:15]: You could have this category:

```suggestion
      category: 'Information',
```

Or this one:

```suggestion
      category: 'Meme',
```

File: src/commands/UCSanDiego.ts
Comment (StormFireFox1) [inline src/commands/UCSanDiego.ts:22]: Maybe there's some merit to sticking this fat string into a separate text file, since it kinda makes the file look wack.

I'll consider it for later, provided I come up with a method to manage long-form strings and descriptions. This is fine for now.

**Review by StormFireFox1 — CHANGES_REQUESTED** [review]
Mini stuff to change, provided we get approval to add this command.

**Comment by trulyronak** [comment]
sadly, this PR has been blocked by the Association for Computing Machinery at the University of California at San Diego :(

**Pull request closed without merging**
